README.md

```diff
@@ -1053,6 +1053,7 @@ Using **CONAN_CLANG_VERSIONS** env variable in Travis ci or Appveyor:
 - **docker_entry_script**: Command to be executed before to build when running Docker.
 - **pip_install**: Package list to be installed by pip before to build. e.j ["foo", "bar"]
 - **docker_32_images**: If defined, and the current build is arch="x86" the docker image name will be appended with "-i386". e.j: "conanio/gcc63-i386"
+- **docker_shell**: Shell command to be executed by Docker. e.j: "/bin/bash -c"
```

Comment:
Document the defaults in windows and linux

Reply:
done